next.config.js

```diff
@@ -3,9 +3,6 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
 })
 
 module.exports = withBundleAnalyzer({
-  future: {
-    webpack5: true,
-  },
```

Comment:
This is not necessary anymore since Next.js 11 opts-in Webpack 5.

Reply:
I saw that in the Next.js migration guide/blog post, but thanks for mentioning it! 😄